### PR TITLE
fix: change select() method to public on text field, number field, and text area

### PR DESCRIPTION
**Pull request by m-akinc**
# Pull Request

## 📖 Description

In #5900 the select() method was added to the text field, number field, and text area, but it was added as a protected member. It should be public so that it can be called by clients.

### 🎫 Issues

See discussion at end of #5626 

## 👩‍💻 Reviewer Notes

N/A

## 📑 Test Plan

N/A

### General

- [x] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [ ] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](/docs/community/code-of-conduct/#our-standards) for this project.

### CODE REVIEW

File: change/@microsoft-fast-foundation-14d4f603-9786-4d4d-8db4-af050205e43e.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```
Comment (chrisdholt) [inline change/@microsoft-fast-foundation-14d4f603-9786-4d4d-8db4-af050205e43e.json:2]: We'll need this to be prerelease in order to keep accurate versioning for our current alphas and betas.

```suggestion
  "type": "prerelease",
```

File: change/@microsoft-fast-foundation-14d4f603-9786-4d4d-8db4-af050205e43e.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  "comment": "change select() method to public on number field, text field, and text area",
+  "packageName": "@microsoft/fast-foundation",
+  "email": "7282195+m-akinc@users.noreply.github.com",
+  "dependentChangeType": "patch"
```
Comment (chrisdholt) [inline change/@microsoft-fast-foundation-14d4f603-9786-4d4d-8db4-af050205e43e.json:6]:
```suggestion
  "dependentChangeType": "prerelease",
```

**Review by chrisdholt — APPROVED** [review]

**Comment by chrisdholt** [comment]
@m-akinc looks like this needs rebase as the API report likely was updated in another PR. Once that's done we'll get this merged right away!

**Comment by m-akinc** [comment]
@chrisdholt I updated and rebased, but didn't get any conflicts to resolve. Let me know if there's still a problem getting this merged.

**Comment by chrisdholt** [comment]
> @chrisdholt I updated and rebased, but didn't get any conflicts to resolve. Let me know if there's still a problem getting this merged.

Thanks, I don't think you'd see any conflicts unless the API report was changed in a way that couldn't be fast forwarded. It's possible that if the build fails again that the API report wasn't able to be merged as it sees your version as "new". In that case, running `yarn` at the root of the repo will generate the report as part of the build process and should result in an updated version. Let's see if the build passes first though!

**Pull request merged**